### ISSUE TITLE
Добавить возможность блокировать конвертацию версии хранилища с пустым комментарием

### ISSUE THREAD

**Issue by mrshadow300373** [1C-Company/GitConverter#147]
Комментарий принимается один единственный раз, при создании отчёта по версиям, и если его нет, то фиксация влетает в гит с комментарием <Комментарий не указан>, и потом уже даже если разработчик спохватился и решил написать, то уже поздно. Переписывать коммиты в гит потом - плохая идея, в особенности если изменения из этой ветки куда то слили. Отсутствие же комментариев усложняет поиск безобразников в коде сводя к минимуму возможности git-blame.

Нужно добавить опцию, останавливающую конвертацию. при которой бы, если комментария к версии нет, не загружать её и все последующие пока он там не появится. 

Далее возможны любые сценарии, например в почту зарегистрированную прислать гневное письмо, об отсутствующем комментарии, либо ничего не делать просто когда выясниться что в гит изменения не прилетели админ попинает разработчика, видя, что его хранилище 1С нет комментариев.

**Comment by marmyshev** [comment]
Была сделана доработка как раз чтобы забивать пустой комментарий текстом <Комментарий не указан>.

Но вот останавливать ли конвертацию, когда не указан - это большой вопрос. Скорее всего это понадобится далеко не всем. 
Я с самой проблемой согласен - что если не указали коммент - это плохо. и плохо потом переписывать историю Git.

Вопрос как дальше обновлять пустые комменты? наверное надо заново запрашивать отчет по версиям для всех пустых.  Если у вас многотысячная история версий в хранилище - заполнять наверное нужно сначала все перед стартом конвертации.

а дальше - уже конроллировать в таком жестоком режиме...

Может быть это сделать в виде расширения? Вполне годная функциональность для расширения.

**Comment by mrshadow300373** [comment]
> Вопрос как дальше обновлять пустые комменты?
Да никак, пока комментарий в хранилище 1С не поставит, не создавать новую версию в справочнике версий, и потом хоть десяток пусть накидает. Если добавит комментарий к той фиксации, которую пропустил, то создать и загрузить эту версию и все последующие, пока не упрёмся в очередной, в котором опять нет комментария, насколько я понял при конвертации каждое помещение в хранилище 1С всё равно обособленно обрабатывается. 
> Скорее всего это понадобится далеко не всем.
Согласен, но вот мне не критично подождать пусть лучше не загрузится в гит конвертере, чем потом улетит в гит. Не долетит до гита, не будет обновления ветки. не будет сборки тестовой базы, в итоге не будет денег. 
> Может быть это сделать в виде расширения?
Просто добавить расширение в новый каталог внутри репо и зареквестить? Или ещё каким то образом?

**Comment by marmyshev** [comment]
> Просто добавить расширение в новый каталог внутри репо и зареквестить? Или ещё каким то образом?

Да, посмотрите как сделаны остальные расширения. И [правила контрибутинга, лицензия](https://github.com/1C-Company/GitConverter/blob/master/CONTRIBUTING.md), создавайте реквест...
